proposals/records.md

```diff
@@ -38,30 +38,48 @@ The synthesized members are as follows:
 
 ### Equality members
 
-Record types produce synthesized implementations for the following methods, where `T` is the
+Record types produce synthesized implementations of the following methods, where `T` is the
 containing type:
+```C#
+public override int GetHashCode();
+public override bool Equals(object other);
+public virtual bool Equals(T other);
+```
+`GetHashCode()` and `Equals(object other)` are overrides of the virtual methods in `System.Object`.
+Any methods on intermediate base classes that would hide those methods are ignored when overriding.
 
-* `object.GetHashCode()` override
-* `object.Equals(object)` override
-* `T Equals(T)` method, where `T` is the current type
-* `Type EqualityContract` get-only property
+Derived record types also override the `Equals(TBase other)` method from each base record type.
 
-If either `object.GetHashCode()` or `object.Equals(object)` are sealed, an error is produced.
+The record type implicitly implements `System.IEquatable<T>` where `T` is the containing type.
+Record types do not re-implement `System.IEquatable<TBase>` for any base type `TBase`.
```

Comment:
>. [](start = 85, length = 1)

What if the user has the record type explicitly derive from `ISomething, System.IEquatable<ISomething>`? Is that implemented? Unless you say so, the answer would be no.